meson.build

```diff
@@ -2760,52 +2765,6 @@ foreach exec : public_programs
         endif
 endforeach
 
-# Enable tests for all supported sanitizers
```

Comment:
Personally I'm OK with removing this (and I agree that it makes sense to use separate ASan/UBSan builds for that). That being said as far as I can remember it was added because there weren't a lot of distributions with separate ASan/UBSan builds downstream.